Document a few more system variables (#6492)
pingcap/docs: system-variables.md

- Default value: `Apache License 2.0`
- This variable indicates the license of your TiDB server installation.

### log_bin

- Scope: NONE
- Default value: `OFF`

> - This variable indicates if [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) is used.

Reviewer: ```suggestion
- This variable indicates whether [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) is used.
```

>
> Unlike in MySQL, the `max_execution_time` system variable currently works on all kinds of statements in TiDB, not only restricted to the `SELECT` statement. The precision of the timeout value is roughly 100ms. This means the statement might not be terminated in accurate milliseconds as you specify.

> ### plugin_dir

Author: @morgo does setting these on a running server make sense? Looks like these should be only used during startup and remain read-only for the rest of the server lifetime.

Reviewer 2: It makes sense. Details are correct, except I would call them "INSTANCE" scoped, not session since they have a getter function that always returns what is in config.

- Scope: SESSION

> - Default value: ""

Author: https://github.com/pingcap/tidb/issues/28084

> - The directory to load plugins from as set by the commandline flag.

Reviewer: ```suggestion
- Indicates the directory to load plugins as specified by a command-line flag.
```

### plugin_load

- Scope: INSTANCE
- Default value: ""

> - Wait for loading of named plugins on startup. Plugins are separated by commas. This is set by a commandline flag.

Reviewer: ```suggestion
- Indicates the plugins to load when TiDB is started. These plugins are specified by a command-line flag and separated by commas.
```

- Default value: ""
- The local unix socket file that the `tidb-server` is listening on when speaking the MySQL protocol.

> ### sql_log_bin

Author: https://github.com/pingcap/tidb/issues/9159

- Scope: SESSION | GLOBAL
- Default value: `ON`

> - This indicates if changes should be written to the [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) or not.

Reviewer: ```suggestion
- Indicates whether to write changes to [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) or not.
```

> **Note:**
>

> > Setting `sql_log_bin` as a global variable is not recommended. Future versions may only allow setting this as a session variable.

Reviewer: ```suggestion
> It is not recommended to set `sql_log_bin` as a global variable because the future versions of TiDB might only allow setting this as a session variable.
```

>
> Unlike in MySQL, the `max_execution_time` system variable currently works on all kinds of statements in TiDB, not only restricted to the `SELECT` statement. The precision of the timeout value is roughly 100ms. This means the statement might not be terminated in accurate milliseconds as you specify.

> ### plugin_dir

Reviewer 3: @TomShawn The scope and description of `plugin_dir` is not consistent with https://github.com/pingcap/docs-cn/pull/11495

- Default value: ""
- The local unix socket file that the `tidb-server` is listening on when speaking the MySQL protocol.

### sql_log_bin

> - Scope: SESSION | GLOBAL

Reviewer 3: @TomShawn Missing "persists to cluster: yes" and "type: boolean"

- This variable returns the current time zone. Values can be specified as either an offset such as '-8:00' or a named zone 'America/Los_Angeles'.
- The value `SYSTEM` means that the time zone should be the same as the system host, which is available via the [`system_time_zone`](#system_time_zone) variable.

### timestamp

- Scope: SESSION
- Default value: ""
- If set to a non-empty value this is the UNIX epoch that is used as timestamp for `CURRENT_TIMESTAMP()`, `NOW()` and other functions. This can be used when restoring or replicating data.

### tmp_table_size

- Scope: SESSION | GLOBAL

> - Default value: 16777216
> - The maximum size of a temporary table in bytes

Reviewer 2: ```suggestion
- Default value: 16777216
- Unit: Bytes
- The maximum size of a temporary table.
```

- This variable returns the current time zone. Values can be specified as either an offset such as '-8:00' or a named zone 'America/Los_Angeles'.
- The value `SYSTEM` means that the time zone should be the same as the system host, which is available via the [`system_time_zone`](#system_time_zone) variable.

### timestamp

- Scope: SESSION
- Default value: ""

> - If set to a non-empty value this is the UNIX epoch that is used as timestamp for `CURRENT_TIMESTAMP()`, `NOW()` and other functions. This can be used when restoring or replicating data.

Reviewer: ```suggestion
- A non-empty value of this variable indicates the UNIX epoch that is used as the timestamp for `CURRENT_TIMESTAMP()`, `NOW()`, and other functions. This variable might be used in data restore or replication.
```

### tmp_table_size

- Scope: SESSION | GLOBAL

> - Default value: 16777216
> - Unit: Bytes
> - The maximum size of a temporary table.

Reviewer: ```suggestion
- Default value: `16777216`
- Unit: Bytes
- Indicates the maximum size of a temporary table.
```

- This variable returns the current time zone. Values can be specified as either an offset such as '-8:00' or a named zone 'America/Los_Angeles'.
- The value `SYSTEM` means that the time zone should be the same as the system host, which is available via the [`system_time_zone`](#system_time_zone) variable.

### timestamp

- Scope: SESSION
- Default value: ""
- A non-empty value of this variable indicates the UNIX epoch that is used as the timestamp for `CURRENT_TIMESTAMP()`, `NOW()`, and other functions. This variable might be used in data restore or replication.

> ### tmp_table_size

Reviewer: This variable is deleted in https://github.com/pingcap/docs/pull/6775.
